mqtt_subscriber/oli_broker.py

```diff
@@ -3,40 +3,64 @@
 import logging
 import traceback
 import paho.mqtt.client as mqtt
+from d3a_api_client.rest_device import RestDeviceClient
+from mqtt_subscriber import allowed_devices_name_mapping, generate_topic_api_client_args_mapping
+from time import time
+
+
+class MQTTBrokerConnectionError(Exception):
+    pass
 
 
 MQTT_PORT = 1883
 MINUTES_IN_HOUR = 60
 MEASUREMENT_PERIOD_MINUTES = 15
+RELOAD_CN_DEVICE_LIST_TIMEOUT_SECONDS = 60 * 15
```

Comment:
```suggestion
RELOAD_CN_DEVICE_LIST_TIMEOUT_SECONDS = 60 * MEASUREMENT_PERIOD_MINUTES
```
Seems quicker to understand the math